internal/exec/engine.go

```diff
@@ -75,6 +81,23 @@ func (e Engine) Run(stageName string) error {
 		return err
 	}
 
+	baseConfigFound := true
+	if err == providers.ErrNoProvider {
+		baseConfigFound = false
+	}
+
+	configExists, err := executil.PathExists(e.ConfigCache)
+	if err != nil {
+		e.Logger.Info("failed to read cached config: %v", err)
+	}
+	// TODO(sohan): This is a hack to emit the journald message only
+	// once for the base config (https://github.com/coreos/ignition/issues/950)
```

Comment:
should we fixup this comment? Is this still considered a hack?

Reply:
IMO yes, because it seems arbitrary to have the presence of the user config cache determine whether the base config should be logged.

Reply:
So we should say something like

```
This is a hack to emit the journald message only once for the base config.
Change this when https://github.com/coreos/ignition/issues/950 is implemented.
```

Reply:
Fixed.